lack/src/test/java/io/datanerds/lack/CachedLackTest.java

```diff
@@ -67,25 +66,13 @@ public void alreadyLocked() throws LackException {
         cachedLack.acquire(resource);
         thrown.expect(LackException.class);
         cachedLack.acquire(resource);
-
-        verify(cache, times(2)).getIfPresent(resource);
```

Comment:
Unfortunately, this code won't get called. The expected exception causes the test to be terminated before those verifications.